pkg/kubectl/cmd/attach.go

```diff
@@ -128,6 +130,9 @@ func (p *AttachOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, argsIn [
 		return cmdutil.UsageError(cmd, fmt.Sprintf("expected fewer than three arguments: POD or TYPE/NAME or TYPE NAME, saw %d: %s", len(argsIn), argsIn))
 	}
 
+	if p.GetPodTimeOut <= 0 {
```

Comment:
Again, helper, which would save changing this, again see: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/util/helpers.go